ui/raidboss/emulator/data/CombatantState.ts

```diff
@@ -1,5 +1,54 @@
-export default class CombatantState {
-  constructor(posX, posY, posZ, heading, targetable, HP, maxHP, MP, maxMP) {
+export interface ICombatantState {
+  posX?: number;
+  posY?: number;
+  posZ?: number;
+  heading?: number;
+  targetable?: boolean;
+  HP?: number;
+  maxHP?: number;
+  MP?: number;
+  maxMP?: number;
+}
+
+// Member names taken from OverlayPlugin's MiniParse.cs
+// Types taken from FFXIV parser plugin
+export interface IPluginState {
```

Comment:
I have to admit I'm not a huge fan of `I` naming.  Could this be `PluginState` or `PluginCombatant`?